docs/redis.md

```
@@ -0,0 +1,126 @@
+---
+title: Redis configuration
+---
+
+# Redis configuration
+
+Queues and workers connect via `useRuntimeConfig().redis` in `resolveConnection()`. Use **`REDIS_*` during dev/build** and **`NUXT_REDIS_*` at runtime** (when running the built server).
+
+Nuxt only applies env overrides at runtime for variables that match declared `runtimeConfig` keys and use the [`NUXT_` prefix](https://nuxt.com/docs/4.x/guide/going-further/runtime-config#environment-variables). After `nuxi build`, your project [`.env` file is not read`](https://nuxt.com/docs/4.x/directory-structure/env#production) — you must set variables in the host environment (Docker `environment:`, Kubernetes secrets, etc.).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)